daml_dit_ddit/subcommand_run.py

```diff
@@ -72,6 +73,7 @@ def subcommand_main(
     env = {
         **os.environ,
         'PYTHONPATH': 'src',
+        'DABL_LEDGER_URL': ledger_url,
```

Comment:
Rather than explicitly specifying a default, I think I'd rather just not specify the URL at all if there is no argument.

Reply:
Sure, that makes sense